impose/__init__.py

```diff
@@ -1,3 +1,7 @@
+import os
+
+os.environ["PYQTGRAPH_QT_LIB"] = "PyQt6"
```

Comment:
This is only important if PyQt5 is installed, correct?

Reply:
I tend to say yes. But pyqtgraph was very thorough in finding pyqt5 and found it even though it was not installed in the ENV running Impose, but in the global python environment. And since in the env only pyqt6 was available running Impose failed then.
I would like to leave this in, just to be sure pyqtraph uses pyqt6, because otherwise it will fail completely.